config.yaml.example

```diff
@@ -41,10 +42,104 @@ colors:
+##  next multiple of this value)
+##
 tabstop: 8
 
-# The editor to be used by the 'e' key (read from $EDITOR by
-# default, but can be overridden here).
+
+##  The editor to be used by the 'e' key (read from $EDITOR by
+##  default, but can be overridden here).
+##
 # editor: "vi"
+
+
+##  Keybindings
+##
+##  The following commands can be mapped:
+##  * display-pager-only     -- Display the pager full-screen
+##  * display-results-only   -- Display the results list full-screen
+##  * split-focus-pager      -- Split screen, focus on pager
+##  * split-focus-results    -- Split screen, focus on results list
+##  * pager-up               -- Scroll one line up in pager
+##  * pager-down             -- Scroll one line down in pager
+##  * pager-pg-up            -- Scroll one page up in pager
+##  * pager-pg-down          -- Scroll one page down in pager
+##  * pager-scroll-left      -- Scroll eight characters left in pager
+##  * pager-scroll-right     -- Scroll eight characters right in pager
+##  * results-up             -- Move to previous result
+##  * results-down           -- Move to next result
+##  * results-pg-up          -- Move one page up in results list
+##  * results-pg-down        -- Move one page down in results list
+##  * prev-result            -- Move to previous result and update pager
+##  * next-result            -- Move to next result and update pager
+##  * pager-goto-result      -- Update pager with currently selected result
+##  * open-file-in-editor    -- Open file in external editor and jump to
+##                              currently selected result
+##  * exit                   -- Exit the application
+##  * unset                  -- Treat keybinding as if not present, fall back to
+##                           -- alternative binding (used to override keybindings)
+##  A command can be mapped to different keys.
+##
+##  Key notation examples
+##  * j                      -- the 'j' key
+##  * J                      -- the 'J' key (Shift-'j')
+##  * Up, Down, Left, Right  -- the corresponding arrow keys
+##  * S-Up, C-Up, M-Up       -- Modifiers Shift, Ctrl, Alt/Meta combined with the
+##                              Up key
+##  * C-M-S-Up, C-S-M-Up     -- Modifiers can be combined, the order is irrelevant
+##                              (both correspond to Control-Alt-Shift-Up)
+##  * The Shift key can only be applied to non-character keys (like Up, Down,
+##    Space, Enter): Shift-'j' is represented by 'J', not by 'S-j'.
+##
+##  Default keybindings:
+keybindings:
+
+    # These keybindings are always in effect, but can be overridden.
+    global-keybindings:
+        q        : exit
+        e        : open-file-in-editor
+
+    # These keybindings apply when navigating the results list. They override
+    # any colliding global-keybinding.
+    results-keybindings:
+        Up       : results-up
+        Down     : results-down
+        PageUp   : results-pg-up
+        PageDown : results-pg-down
```

Comment:
`pg` looks cryptic and saves two characters over `page`, I don’t think this abbreviation should be there